README.md

```diff
@@ -71,17 +71,8 @@ app.get(
 
 ### Login
 
-The login route will redirect to a [WorkOS OAuth 2.0 authorization URL](https://workos.com/docs/reference/sso/authorize/get). When redirecting to this route, be sure to include one of the [supported query parameters](https://workos.com/docs/reference/sso/authorize/get#authorize-get-parameters)
-
-> **Note**
-> An additional `email` query parameter is supported which will extract the `domain` and forward it to WorkOS
-
-**Example**
-
-```
-location.href = "/auth/workos/login?domain=gmail.com"
-```
+The login route will redirect to a [WorkOS OAuth 2.0 authorization URL](https://workos.com/docs/reference/sso/get-authorization-url). When redirecting to this route, be sure to include one of the [supported query parameters](https://workos.com/docs/reference/sso/get-authorization-url)
```

Comment:
The links to the WorkOS Docs have been updated, as there were some changes with the release of the new Docs.